### PR TITLE
Add install option to skip rtx install

### PR DESCRIPTION
**Pull request by higebu**
I'd like to add `install` option to skip `rtx install`.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ inputs:
   tool_versions:
     required: false
     description: If present, this value will be written to the .tool-versions file
+  install:
+    required: false
+    default: 'true'
```
Comment (jdx) [inline action.yml:13]: Shouldn't this just be `true`?

**Review by jdx — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (higebu) [reply action.yml:13]: According to the GitHub document, the value should be `string` .
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

File: src/main.ts [hunk 2]
```diff
@@ -13,7 +13,10 @@ async function run(): Promise<void> {
   await setupRTX()
   await setEnvVars()
   await exec.exec('rtx', ['--version'])
-  await exec.exec('rtx', ['install'])
+  const install = core.getInput('install', {required: false})
+  if (install === 'true') {
```
Comment (jdx) [inline src/main.ts:17]: If I'm reading the docs right, the default is a "string representing the value", which means the default would be `true` not `'true'`. Either way, I would just make this check for truthiness so it works regardless
```suggestion
  if (install) {
```

**Review by jdx — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (higebu) [reply action.yml]: I changed this to `true`.

File: src/main.ts (hunk 2 above)
Comment (higebu) [reply src/main.ts]: I applied your suggestion. Thank you.

**Pull request merged**